### ISSUE TITLE
Validate that octets and list fields are the last in the structure

### ISSUE THREAD

**Issue by cmebarrow** [reaktivity/nukleus-maven-plugin#29]
Since ListFW and OctetsFW are currently unbounded, structures containing a list or octets field which is not the last field in the structure will not work. For example, generating the following IDL actually creates a flyweight with compilation errors:
```
    struct FlatWithList
    {
        list<string> list1;
        uint64 fixed1;
    }
```
We should validate in the grammar that list and octet fields are last.

**Comment by cmebarrow** [comment]
This is being addressed in PR #30

**Comment by cmebarrow** [comment]
Closing since PR #30 has been merged
